From 7f1f1bd381ddbf76e9645131cec41806dd2c301b Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Thu, 3 Mar 2016 06:24:24 -0500
Subject: [PATCH] wr-policy: allow mount_t getattr of all filesystems

Reference sources: selinux-policy-3.13.1-128.21.fc22.src.rpm
Changes come from: policy-f22-base.patch

Fix avc denial:

  avc: denied { getattr } for pid=1720 comm="umount" \
  name="/" dev="tmpfs" ino=65541 \
  scontext=system_u:system_r:mount_t:s0-s15:c0.c1023 \
  tcontext=system_u:object_r:tmpfs_t:s0 \
  tclass=filesystem permissive=0

  avc: denied { getattr } for pid=1722 comm="umount" \
  name="/" dev="tracefs" ino=1 \
  scontext=system_u:system_r:mount_t:s0-s15:c0.c1023 \
  tcontext=system_u:object_r:unlabeled_t:s15:c0.c1023 \
  tclass=filesystem permissive=0

Upstream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 policy/modules/system/mount.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/mount.te b/policy/modules/system/mount.te
index d31406a..122f375 100644
--- a/policy/modules/system/mount.te
+++ b/policy/modules/system/mount.te
@@ -111,6 +111,7 @@ fs_getattr_tmpfs(mount_t)
 fs_getattr_rpc_pipefs(mount_t)
 fs_getattr_cifs(mount_t)
 fs_getattr_nfs(mount_t)
+fs_getattr_all_fs(mount_t)
 fs_mount_all_fs(mount_t)
 fs_unmount_all_fs(mount_t)
 fs_remount_all_fs(mount_t)
-- 
2.7.4

